app/views/shared/_post.html.erb

```diff
@@ -4,60 +4,63 @@
 <div class="row">
 
   <div class="col-md-8 lead">
-    <div class="panel panel-default panel-info break-word">
+    <div class="panel panel--default panel-info break-word">
```

Comment:
According to BEM conventions this should be `panel panel--default panel--info break-word`.

If ` break-word` is an utility class I would use ` u-break-word`.

Reply:
Some css classes were already there:

![image](https://user-images.githubusercontent.com/3697530/60496741-bb7b2680-9cb3-11e9-87c7-f456909aba7a.png)

maybe we should clean and improve some html in a future revision